test/ttmlir/Dialect/TTNN/multiple_add_with_loc_input_layout_override.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="enable-optimizer=true sharding-pass-enabled=true resharding-enabled=true insert-reshard=add_0_1_2=0" %s | FileCheck %s
```

Comment:
Should this be insert-reshard=add_0_1_2:0 ?

Reply:
Also add one more test which will call only optimizer and use new override. See test_grid_set.mlir